### PR TITLE
more easy to change mode，and add `z` to undo

### PR DESCRIPTION
**Pull request by StevenJokes**
Inspired by my issue https://github.com/microsoft/vscode-python/issues/13580

### CODE REVIEW

File: docs/python/jupyter-support.md [hunk 1]
```diff
@@ -148,6 +148,10 @@ Deleting a code cell can be accomplished by hovering over a code cell and using
 
 ![Delete a code cell](images/jupyter/native-code-cells-06.png)
 
+### Undo what you done last
```
Comment (JoshuaPartlow) [inline docs/python/jupyter-support.md:151]:
```suggestion
### Undo your last change
```

File: docs/python/jupyter-support.md [hunk 2]
```diff
@@ -148,6 +148,10 @@ Deleting a code cell can be accomplished by hovering over a code cell and using
...
+
+We can use `kbstyle(z)` to undo what you done last, for example, you can have your carelessness deleted cell be undone.
```
Comment (JoshuaPartlow) [inline docs/python/jupyter-support.md:153]:
```suggestion
You can use the `kbstyle(z)` key to undo your previous change, for example, if you've made an accidental edit you can undo it to the previous correct state. 
```

**Comment by JoshuaPartlow** [comment]
Perfect, thanks very much.

**Comment by StevenJokes** [comment]
.. I just want to add more.

**Comment by StevenJokes** [comment]
You merge too quickly. I'm adding more. Then I found I can't edit....
https://github.com/microsoft/vscode-docs/pull/3962

**Comment by JoshuaPartlow** [comment]
Sorry about that. Does the other PR have everything you wanted?

**Pull request merged**